src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -221,9 +221,9 @@ object AddDatasetMetadataToDeposit {
     dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues => {
       val srsName = createSrsName(mdKeyValues)
       // coordinate order x, y = longitude (DCX_SPATIAL_X), latitude (DCX_SPATIAL_Y)
-      val xy = s"${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")}"
+      lazy val xy = s"${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")}"
       // coordinate order y, x = latitude (DCX_SPATIAL_Y), longitude (DCX_SPATIAL_X)
-      val yx = s"${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")}"
+      lazy val yx = s"${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")}"
```

Comment:
after seeing it again, it might be better to calculate the value for `DCX_SPATIAL_X` and `DCX_SPATIAL_Y` separately (_don't repeat yourself_) and then compose the two values in the correct string inside the pattern match below. You did this already below at line 267-272. Note: in this case you don't need the lazy here, since you use `DCX_SPATIAL_X` and `DCX_SPATIAL_Y` in both cases.